exchanges/bittrex/bittrex_test.go

```diff
@@ -726,3 +726,11 @@ func TestGetHistoricCandlesExtended(t *testing.T) {
 		t.Fatal(err)
 	}
 }
+func TestWsConn(t *testing.T) {
+	t.Parallel()
+	err := b.Websocket.Connect()
+	if err != nil {
+		t.Fatal(err)
+	}
+	time.Sleep(time.Second * 20)
```

Comment:
Please update this test to adequately test the function

Reply:
It's my mistake. I forgot to remove it.
Thank you